lib/omniauth/strategies/google-oauth2-access-token.rb

```diff
@@ -80,6 +80,10 @@ def request_phase
         form.to_response
       end
 
+      def credentials
+        { token: self.access_token.token }
```

Comment:
is the `token` method always defined? will access_token ever be nil?

LGTM otherwise